traitsui/wx/editor.py

```diff
@@ -61,17 +61,6 @@ def update_editor(self):
         if self.control.GetValue() != new_value:
             self.control.SetValue(new_value)
 
-    def error(self, excp):
-        """Handles an error that occurs while setting the object's trait value."""
-        dlg = wx.MessageDialog(
-            self.control,
-            str(excp),
-            self.description + " value error",
-            wx.OK | wx.ICON_INFORMATION,
```

Comment:
information from pyface will now also have `| wx.STAY_ON_TOP` for the style.

Reply:
I think that's good in this case.